sinonjs/sinon
# Tidy up docs

This PR tidies up some of the source for the documentation site and fixes #1573 in the process

#### How to verify - mandatory
1. Check out this branch
1. `cd docs`
1. `bundle exec jekyll serve`
1. Navigate to http://localhost:4000/releases/v4.0.0/
1. Observe the menu item "Guides"
1. Observe that the page has links to each of the migration guides
1. Observe that this partial is used for all the existing releases, as well as in `release-source`

![2017-09-30 at 12 01](https://user-images.githubusercontent.com/20321/31044826-359ff2ea-a5d7-11e7-8d78-cff7298b5dc9.png)
